outputs.tf

```diff
@@ -39,6 +39,11 @@ output "kubeconfig" {
   value       = "${data.template_file.kubeconfig.rendered}"
 }
 
+output "kubeconfig_filename" {
+  description = "The filename of the generated kubectl config."
+  value       = "${local_file.kubeconfig.*.filename}"
```

Comment:
If `kubeconfig` is disabled (`var.write_kubeconfig = 0`) this value won't work. Change this to:
`value = ${element(concat(local_file.kubeconfig.*.filename, list("")), 0)}`

Reply:
Thanks for the suggestion, @antonbabenko .
I updated the PR.